aamt_parse2.pm

```diff
@@ -482,10 +734,11 @@ sub pars_PublishSite
             $strYesOrNo =" ";
             while($strYesOrNo!~/^\s*[YynN]\s*$/)
             {
-                ilog_printf(1, "    Would you like to retry publishing the site [$strSiteName]? (Y/N):");
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can get the information from the cmd line, else keep the current logic of asking the user for input.